packages/browser/package.json

```diff
@@ -61,11 +61,5 @@
     "build-node": "BROWSER_PKG=1 pnpm run --filter rolldown build-node",
     "preinstall": "npx only-allow pnpm"
   },
-  "dependencies": {
-    "@napi-rs/wasm-runtime": "^0.2.8",
```

Comment:
Why remove this?

Reply:
`@rolldown/browser` is using the `package.json` of `packages/browser`. It requires to be in `dependecies`, when users install `@rolldown/browser`.

Reply:
Thanks Yunfei! Is it only `@napi-rs/wasm-runtime`? I have restored it now.

Reply:
I think so. We could release test version to see if everything works as expected.

Reply:
OK sounds good.

Maybe we could consider doing something for rolldown packages like Knip does: publish with [pkg.pr.new](https://github.com/stackblitz-labs/pkg.pr.new) and then a real installation (just like from the npm registry) in a real repository. Example: https://github.com/webpro-nl/knip/blob/main/.github/workflows/integration.yml

Reply:
Yes. We had set up releasing process for pkg.pr.new. I'll take a look tomorrow, or boshen might do it. It's midnight in China. I need to go to bed.